Traceback (most recent call last):
  File "c:\Users\Personal\OneDrive\My Projects\Automating The Boring Stuff(Python)\Debugging\log.py", line 5, in <module>
    raise Exception('This is the error message.')
Exception: This is the error message.
